chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+as observations can be aggregated and linked consistently,
+and easier to clean, as each attribute only needs to be checked once.
+
+% An example of untidy data
+A very simple way for our survey example to became untidy is to include
+household-level variables in the same data set,
+such as whether the household has a refrigerator.
+In this case, some attributes would be measured at
+household level, and others at household member level,
+and it would not be possible for each column to represent a variable
+\textit{and} for each row to represent an observation \textit{at the same time}.
+Either the household-level variable would be repeated for all its members,
+or member-level variables would be stored in 
+\textbf{wide format}.\sidenote{\textbf{Wide data:}
+	a table where a single variable is divided into multiple columns,
+	for example one for each individual.
+}
+An alternative example is a dataset on transactions that also contains information
+about the parts involved in each transaction on the same table.
+The next two sections will outline steps to check if your data is tidy,
+and to tidy it if it is not.
+
+\subsection{Tidying raw data}
+
+% Is the uniquely identified data tidy?
+Sometimes, making sure that data is uniquely and fully identified 
+is enough to yield a tidy dataset.
+This happens when a data source or survey instrument
+contains a single unit of observation.
+If you are working with data in this format,
+you may skip the tidying process,
+which can become quite intricate.
+In this case the cleaning process will result in a single dataset.
+
+% Examples of untidy raw data
+The most common case of untidy raw data encountered in development research
+are datasets consisting of multiple units of observation stored in the same table. 
+If your rows include multiple nested observational units,
+having an identifying variable for each row doesn't mean having an identifier for each observation.
+This is the case of our previous example of untidy data:
+a survey that collects both household and household-member level information.
+In this case, 
+the easiest way to handle the data is often to break it into two tidy datasets:
+one at the household level, and one at the member level.
+You will use these separate datasets for the rest of the data cleaning process.
+
+% How to tidy data
+Creating these two tidy datasets will involve a few different transformations.
+Many common survey software export \textbf{wide datasets},
+where each row represents the highest level of observation,
+and variables observed in nested units are separated into one column per sub-unit,
+such as \texttt{age\_1}, \texttt{gender\_1}, \texttt{age\_2}, \texttt{gender\_2}, etc.
+To create tidy datasets from this data,
+you first need to identify which variables were observed at which levels.
+In our example, the presence of a refrigerator was collected at household level (the higher unit),
+while age and gender where observed at the member sub-unit.
```

Comment:
We are almost in the end of this section but only here are we uncovering the core of what is the underlying reason why tidy is superior